public/style/_icons.scss

```diff
@@ -6,153 +6,150 @@
 	top: 2px;
 }
 
-.i-add-image {
+.i-public--images--icons--add-image {
 	@extend %svg-common;
-	background-position: 25.806451612903224% 0;
+	background-position: 0 0;
 	width: 16px;
 	height: 16px;
 }
 
-.i-add-image-grey {
+.i-public--images--icons--add-image-grey {
 	@extend %svg-common;
-	background-position: 77.41935483870968% 26.229508196721312%;
+	background-position: 25.806451612903224% 0;
 	width: 16px;
 	height: 16px;
 }
 
-.i-article {
+.i-public--images--icons--article {
 	@extend %svg-common;
 	background-position: 0 26.229508196721312%;
 	width: 16px;
 	height: 16px;
 }
 
-.i-clock {
+.i-public--images--icons--clock {
 	@extend %svg-common;
 	background-position: 25.806451612903224% 26.229508196721312%;
 	width: 16px;
 	height: 16px;
 }
 
-.i-cross {
+.i-public--images--icons--cross {
 	@extend %svg-common;
 	background-position: 51.61290322580645% 0;
 	width: 16px;
 	height: 16px;
 }
 
-.i-cross-red {
+.i-public--images--icons--cross-red {
 	@extend %svg-common;
-	background-position: 98.46153846153847% 64.0625%;
+	background-position: 98.46153846153847% 43.75%;
 	width: 13px;
 	height: 13px;
 }
 
-.i-cross-white {
+.i-public--images--icons--cross-white {
 	@extend %svg-common;
-	background-position: 98.46153846153847% 43.75%;
+	background-position: 98.46153846153847% 64.0625%;
 	width: 13px;
 	height: 13px;
 }
 
-.i-delete {
```

Comment:
@rhystmills I think this has broken the delete icon on parent tags, this should appear in the right hand column here:

<img width="546" alt="Screenshot 2022-10-05 at 11 06 00" src="https://user-images.githubusercontent.com/2619836/194035713-8929c54e-dc0d-49f5-9913-26d5f6bd7953.png">